javascript/javascript1/week2/index.js

```diff
@@ -0,0 +1,60 @@
+
+
+
+function getFullName(firstName, surName, formalName){
+    if(formalName===true){
+        return 'Lord!' +' ' + firstName +' ' +surName;
+    } else{
+         return firstName +' ' + surName;
+    }
+};
+console.log(getFullName('', '', ''));
+
+
+//Event application
+
+function weekdayEvent(event) {
+let weekDays= ['Sunday','Monday','Tursday','Wedensday','Thursday', 'Friday', 'Sutarday'];
```

Comment:
THink of using `const` here
```suggestion
const weekDays= ['Sunday','Monday','Tursday','Wedensday','Thursday', 'Friday', 'Sutarday'];
```